public/locales/en/settings.json

```diff
@@ -1,5 +1,12 @@
 {
   "title": "Settings",
+  "save": "Save",
+  "saving": "Saving…",
+  "reset": "Reset",
+  "pinningServices": {
+    "title": "Pinning Services",
+    "description": "Support for third-party remote pinning services will be added in a future release, and you'll be able to add and configure them here. In the meantime, use local pinning to ensure files on your local node persist and are never garbage-collected."
```

Comment:
How about this? Makes it feel a little more like insurance, i.e. "pin stuff you really want to keep just in case you accidentally delete it from MFS."

```suggestion
    "description": "Support for third-party remote pinning services will be implemented in a future release, and you'll be able to add and configure them here. In the meantime, use local pinning when you want to ensure an item on your node is never garbage-collected, even if you remove it from Files."
```